### PR TITLE
Removed unwanted features

### PR DESCRIPTION
**Pull request by subash6465**

### CODE REVIEW

File: app/api/README.md
Comment (varsharamanujam) [inline app/api/README.md:1]: make this same as main branch

File: app/api/meeting.js
Comment (varsharamanujam) [inline app/api/meeting.js:1]: make this same as main branch

File: app/api/meeting/meeting.js
Comment (varsharamanujam) [inline app/api/meeting/meeting.js:1]: make this same as main branch

File: app/ssl/README.md
Comment (varsharamanujam) [inline app/ssl/README.md:1]: delete this file

**Review by varsharamanujam — CHANGES_REQUESTED** [review]
1. change occurrence of videolify to Iproctor
2. delete readme in ssl folder 
3. Make files which you haven't made any changes same as the one in main branch

**Comment by subash6465** [comment]
check now

**Pull request closed without merging**